packages/aws-cdk-lib/aws-rds/lib/instance-engine.ts

```diff
@@ -771,22 +792,45 @@ export class MysqlEngineVersion {
    * @deprecated MySQL 5.7.36 is no longer supported by Amazon RDS.
    */
   public static readonly VER_5_7_36 = MysqlEngineVersion.of('5.7.36', '5.7');
-  /** Version "5.7.37". */
+  /**
+   * Version "5.7.37"
+   * @deprecated MySQL 5.7.37 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_37 = MysqlEngineVersion.of('5.7.37', '5.7');
-  /** Version "5.7.38". */
+  /**
+   * Version "5.7.38"
+   * @deprecated MySQL 5.7.38 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_38 = MysqlEngineVersion.of('5.7.38', '5.7');
-  /** Version "5.7.39". */
+  /**
+   * Version "5.7.39"
+   * @deprecated MySQL 5.7.39 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_39 = MysqlEngineVersion.of('5.7.39', '5.7');
-  /** Version "5.7.40". */
+  /**
+   * Version "5.7.40"
+   * @deprecated MySQL 5.7.40 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_40 = MysqlEngineVersion.of('5.7.40', '5.7');
-  /** Version "5.7.41". */
+  /**
+   * Version "5.7.41"
+   * @deprecated MySQL 5.7.41 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_41 = MysqlEngineVersion.of('5.7.41', '5.7');
-  /** Version "5.7.42". */
+  /**
+   * Version "5.7.42"
+   * @deprecated MySQL 5.7.42 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_42 = MysqlEngineVersion.of('5.7.42', '5.7');
-  /** Version "5.7.43". */
+  /**
+   * Version "5.7.43"
+   * @deprecated MySQL 5.7.43 is no longer supported by Amazon RDS.
+   */
   public static readonly VER_5_7_43 = MysqlEngineVersion.of('5.7.43', '5.7');
   /** Version "5.7.44". */
   public static readonly VER_5_7_44 = MysqlEngineVersion.of('5.7.44', '5.7');
+  /** Version "5.7.44-rds.20240408". */
+  public static readonly VER_5_7_44_RDS_20240408 = MysqlEngineVersion.of('5.7.44-rds.20240408', '5.7');
```

Comment:
See [announcement](https://aws.amazon.com/about-aws/whats-new/2024/05/amazon-rds-sql-extended-support-minor-5-7-44-rds-20240408/)

Reply:
Seems like there is extended support for [version 5.7.44-RDS.20240529](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html#mysql-extended-support-releases-version-5.7.44-20240529:~:text=5.7.44%2DRDS.20240408-,RDS%20Extended%20Support%20for%20RDS%20for%20MySQL%20version%205.7.44%2DRDS.20240529,-RDS%20Extended%20Support) as well now, could we update this PR to include the latest versions?

Reply:
Thanks for letting me know, I've updated them 👍